### ISSUE TITLE
[BridgeBidding] Check random action

### ISSUE THREAD

**Issue by harukaki** [sotetsuk/pgx#730]

**Comment by harukaki** [comment]
1並列, 4並列で共に、stateのsvg表示, action, current_player, rewardの値は全て手元の計算と一致した。
一方で、shuffled_playersでの並び順が0, 1のチームがNS, 2, 3のチームがEWに固定されてしまっている。

**Comment by harukaki** [comment]
> 

https://github.com/sotetsuk/pgx/blob/4d541423fe28cb65f2690ecd1afc2f13f6fe0560/pgx/bridge_bidding.py#L205
この部分がおかしい
`jax.random.randint(rng, (1,), 1, 2)[0])`->`jax.random.randint(rng, (1,), 0, 2)[0])`

**Comment by harukaki** [comment]
1並列、4並列に置いて目視での確認では問題ない。
